services/QuillLMS/lib/tasks/backfill.rake

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+namespace :backfill do
+  # rake 'backfill:updated_at[ClassroomUnit "2020-01-01"]'
+  desc 'backfills updated_at so null values do not exist'
+  task :updated_at, [:klass, :fill_datetime] => :environment do |t, args|
+    klass = args[:klass].constantize
+    fallback_fill_datetime = DateTime.new(2018,4,1)
```

Comment:
This date seems arbitrary. Might be worth removing this fallback. Then if a fallback isn't passed in, it just doesn't update if there isn't a `created_at`.

Reply:
Fortunately, for the tables in question, all records have non-nil `created_at` values